ChangeLog.zephyr.md

```diff
@@ -3,6 +3,9 @@
 Need to take care to not break these changes when updating pico-sdk.
 
 ## Patch List:
+  - [#11] pico-sdk: Disabling sanity check the IRQ status
```

Comment:
please refrain from using "sanity check" term https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#rule-a-2-inclusive-language

Reply:
Thank you for pointing it out.
The changes made during the rebase were not reflected, resulting in inconsistent content.
So, I changed the entire sentence.